Lib/fontTools/merge.py

```diff
@@ -505,7 +505,15 @@ def merge(self, m, tables):
 	assert len(tables) == len(m.duplicateGlyphsPerFont)
 	for i,(table,dups) in enumerate(zip(tables, m.duplicateGlyphsPerFont)):
 		if not dups: continue
-		assert (table is not None and table is not NotImplemented), "Have duplicates to resolve for font %d but no GSUB" % (i + 1)
+		if table is None or table is NotImplemented:
+			# Checks whether the dups are equivalent or not.
+			# Discard gid if its shape is not equal to that of oldgid.
+			for oldgid, gid in dups.iteritems():
```

Comment:
iteritems() no longer exists on python3, please just use `items()` for py2.py3 compatibility.